src/main/scala/org/broadinstitute/dsde/firecloud/webservice/TrialApiService.scala

```diff
@@ -20,37 +20,51 @@ trait TrialApiService extends HttpService with PerRequestCreator with FireCloudD
   private implicit val executionContext = actorRefFactory.dispatcher
   val trialServiceConstructor: () => TrialService
 
-  // TODO: See if it makes sense to use DELETE for terminate, and perhaps PUT for disable
   val trialApiServiceRoutes: Route = {
-    pathPrefix("trial" / "manager") {
-      post {
-        path("enable|disable|terminate".r) { (operation: String) =>
-          requireUserInfo() { managerInfo => // We will need the manager's credentials for authentication downstream
-            entity(as[Seq[String]]) { userEmails => requestContext =>
-                perRequest(requestContext,
-                  TrialService.props(trialServiceConstructor),
-                  updateUsers(managerInfo, TrialOperations.withName(operation), userEmails))
-            }
+    pathPrefix("trial") {
+      path("userAgreement") {
+        put {
+          requireUserInfo() { userInfo =>
+            requestContext =>
+              perRequest(requestContext,
+                TrialService.props(trialServiceConstructor),
+                TrialService.RecordUserAgreement(userInfo))
           }
-        } ~
-        path("projects") {
-          parameter("count".as[Int] ? 0) { count =>
-            parameter("operation") { op =>
-              requireUserInfo() { userInfo => requestContext =>
-                val message = op.toLowerCase match {
-                  case "create" => Some(TrialService.CreateProjects(userInfo, count))
-                  case "verify" => Some(TrialService.VerifyProjects(userInfo))
-                  case "count" => Some(TrialService.CountProjects(userInfo))
-                  case "report" => Some(TrialService.Report(userInfo))
-                  case _ => None
+        }
+      } ~
+      pathPrefix("manager") {
+        // TODO: See if it makes sense to use DELETE for terminate, and perhaps PUT for disable
+        post {
+          path("enable|disable|terminate".r) { (operation: String) =>
+            requireUserInfo() { managerInfo => // We will need the manager's credentials for authentication downstream
+              entity(as[Seq[String]]) { userEmails =>
+                requestContext =>
+                  perRequest(requestContext,
+                    TrialService.props(trialServiceConstructor),
+                    updateUsers(managerInfo, TrialOperations.withName(operation), userEmails))
+              }
+            }
+          } ~
+            path("projects") {
```

Comment:
Nit: the block `48`-`67` is indented one level too far

Reply:
Thanks for the catch!